### PR TITLE
add user file

### PR DESCRIPTION
**Pull request by lilachic**
A new file to add the repository

### CODE REVIEW

File: lilachic.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+My favourite text placeholder Lorem Ipsum
+
```
Comment (plaihonen) [inline lilachic.txt:2]: Is this space needed?

Comment (plaihonen) [reply lilachic.txt:2]: To specify, is the blank line needed?

Comment (lilachic) [reply lilachic.txt:2]: Not needed,  but to show clearly the link.

Comment (plaihonen) [reply lilachic.txt:2]: Allrighty then.....

**Pull request merged**